src/components/QuestionPage.js

```diff
@@ -58,43 +66,62 @@ class QuestionPage extends Component {
       if (this.state.answer == this.state.correctAnswer) {
         return {
           backgroundColor: '#58E778',
-          height: 32,
           width: 294,
           justifyContent: 'center',
           alignItems: 'center',
-          marginBottom: 21,
+          marginBottom: 10,
           flexDirection: 'row',
         };
       } else {
         return {
-          backgroundColor: '#F16B8B',
-          height: 32,
+          backgroundColor: '#ec4646',
           width: 294,
           justifyContent: 'center',
           alignItems: 'center',
-          marginBottom: 21,
+          marginBottom: 10,
           flexDirection: 'row',
         };
       }
     } else {
-      return {
-        backgroundColor: '#6BB1F1',
-        height: 32,
-        width: 294,
-        justifyContent: 'center',
-        alignItems: 'center',
-        marginBottom: 21,
-        flexDirection: 'row',
-      };
+      if (
+        item == this.state.disableAnswer1 ||
+        item == this.state.disableAnswer2
+      ) {
+        return {
+          backgroundColor: '#a6a9b6',
+          width: 294,
+          justifyContent: 'center',
+          alignItems: 'center',
+          marginBottom: 10,
+          flexDirection: 'row',
+        };
+      } else {
+        return {
+          backgroundColor: '#6BB1F1',
+          width: 294,
+          justifyContent: 'center',
+          alignItems: 'center',
+          marginBottom: 10,
+          flexDirection: 'row',
+        };
+      }
     }
   }
-  disable() {
+  disable(item) {
     var True = True;
     if (this.state.answer !== 0) {
       return {
         True,
       };
     }
+    if (
+      item == this.state.disableAnswer1 ||
+      item == this.state.disableAnswer2
```

Comment:
Bu iki durum tek if de olur gibi

Reply:
anlamadım olayı ama eğer kast ettiğin bu ise;(alttaki duruma getirilmesi)

```if ([this.state.disableAnswer1, this.state.disableAnswer2].includes(item)) {
      return true;
    }```


bu şekilde değiştirdim